test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare module 'test-case-reporting' {
+  /**
+   * A standard logging interface.
+   */
+  export interface Logger {
```

Comment:
This was added for Probot (GitHub) apps, because Probot provides app- and request-level loggers with this interface, which matches that of `console.XXX`, so this type definition allowed us to use a single uniform `Logger` interface everywhere. Since our app will just use regular `console.` logging, this isn't necessary here

Reply:
Done.